openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -1,268 +1,35 @@
-        data["instance_id"] = tool.Name
-
-        return data
-
-    def get_pre_create_attr_defs(self):
-        """Settings for create page"""
-        attr_defs = [
-            self._get_render_target_enum(),
-            self._get_reviewable_bool(),
-            self._get_frame_range_enum(),
-            self._get_image_format_enum(),
-        ]
-        return attr_defs
-
-    def get_instance_attr_defs(self):
-        """Settings for publish page"""
-        return self.get_pre_create_attr_defs()
-
-    def pass_pre_attributes_to_instance(self, instance_data, pre_create_data):
-        creator_attrs = instance_data["creator_attributes"] = {}
-        for pass_key in pre_create_data.keys():
-            creator_attrs[pass_key] = pre_create_data[pass_key]
-
-    # These functions below should be moved to another file
-    # so it can be used by other plugins. plugin.py ?
-    def _get_render_target_enum(self):
-        rendering_targets = {
-            "local": "Local machine rendering",
-            "frames": "Use existing frames",
-        }
-        if "farm_rendering" in self.instance_attributes:
-            rendering_targets["farm"] = "Farm rendering"
-
-        return EnumDef(
-            "render_target", items=rendering_targets, label="Render target"
-        )
-
-    def _get_frame_range_enum(self):
-        frame_range_options = {
-            "asset_db": "Current asset context",
-            "render_range": "From render in/out",
-            "comp_range": "From composition timeline",
-        }
-
-        return EnumDef(
-            "frame_range_source",
-            items=frame_range_options,
-            label="Frame range source",
-        )
 
-    def _get_reviewable_bool(self):
-        return BoolDef(
-            "review",
-            default=("reviewable" in self.instance_attributes),
-            label="Review",
-        )
+    def get_detail_description(self):
+        return """Fusion Saver to generate image sequence.
 
-    def _get_image_format_enum(self):
-        image_format_options = ["exr", "tga", "tif", "png", "jpg"]
-        return EnumDef(
-            "image_format",
-            items=image_format_options,
-            default=self.image_format,
-            label="Output Image Format",
-        )
+        This creator is expected for publishing of image sequences for 'render'
+        product type. (But can publish even single frame 'render'.
 
-    def apply_settings(self, project_settings):
-        """Method called on initialization of plugin to apply settings."""
+        Select what should be source of render range:
+        - "Current asset context" - values set on Asset in DB (Ftrack)
+        - "From render in/out" - from node itself
+        - "From composition timeline" - from timeline
 
-        # plugin settings
-        plugin_settings = project_settings["fusion"]["create"][
-            self.__class__.__name__
-        ]
+        Supports local and deadline rendering.
```

Comment:
Even though I like the clarity - I wonder whether we want to name "deadline" explicitly here with the goal being to get addons more and more separated?